extensions/ql-vscode/src/vscode-tests/cli-integration/run-cli.test.ts

```diff
@@ -68,4 +70,16 @@ describe('Use cli', function() {
     const queryInfo: QueryInfoByLanguage = await cli.resolveQueryByLanguage(getOnDiskWorkspaceFolders(), Uri.file(queryPath));
     expect((Object.keys(queryInfo.byLanguage))[0]).to.eql('javascript');
   });
+
+  it.only('should resolve printAST queries', async function() {
+    skipIfNoCodeQL(this);
+
+    const result = await resolveQueries(cli, {
+      dbschemePack: 'codeql/javascript-all',
```

Comment:
How do these tests get the query repo checkout? If we're splitting over multiple CLI versions, we should use the corresponding `codeql-cli/version` tags from the query repos, as well as latest `main` for the last released CLI.

Reply:
As discussed, the integration tests checks out the expected codeql version for each CLI version.